Mathlib/Algebra/Order/Hom/Basic.lean

```diff
@@ -139,7 +137,6 @@ theorem le_map_add_map_div [Group α] [AddCommSemigroup β] [LE β] [MulLEAddHom
     (a b : α) : f a ≤ f b + f (a / b) := by
   simpa only [add_comm, div_mul_cancel'] using map_mul_le_add f (a / b) b
 #align le_map_add_map_div le_map_add_map_div
--- #align le_map_add_map_sub le_map_add_map_sub -- Porting note: TODO: `to_additive` clashes
```

Comment:
Why was this removed?

Reply:
Because the current mathport output for the file has that exact same line in the exact same place.

Reply:
I don't understand what you mean, can you elaborate?

Reply:
With that line uncommented I get this error message:
`le_map_add_map_sub has already been aligned (to le_map_add_map_sub)`
Since `to_additive` has been improved to the point where it can take care of this all by itself, I feel this doesn't warrant a porting note anymore.

Reply:
Ah nevermind, I understand what's happening here now (and it's not what you described).

Because this is `to_additive existing`, there is a line in another file that is manually doing the correct alignment.